lib/transaction/transaction.js

```diff
@@ -374,15 +374,15 @@ Transaction.prototype.fromObject = function fromObject(arg) {
   _.each(transaction.outputs, function(output) {
     self.addOutput(new Output(output));
   });
+  if (transaction.fee) {
+    this.fee(transaction.fee);
```

Comment:
Should we use `this._fee = transaction.fee` so that `_updateChangeOutput()` isn't called at all?